pkg/shares/utils.go

```diff
@@ -16,39 +13,12 @@ func DelimLen(size uint64) int {
 	return binary.PutUvarint(lenBuf, size)
 }
 
-func isPowerOf2(v uint64) bool {
-	return v&(v-1) == 0 && v != 0
-}
-
-func BlobsFromProto(blobs []core.Blob) ([]coretypes.Blob, error) {
-	result := make([]coretypes.Blob, len(blobs))
-	for i, blob := range blobs {
-		if blob.ShareVersion > math.MaxUint8 {
-			return nil, fmt.Errorf("share version %d is too large to be a uint8", blob.ShareVersion)
-		}
-		result[i] = coretypes.Blob{
-			NamespaceID:  blob.NamespaceId,
-			Data:         blob.Data,
-			ShareVersion: uint8(blob.ShareVersion),
-		}
-	}
-	return result, nil
-}
```

Comment:
Note: these functions are either not used (and probably shoudn't be) or redundant